doc/services/zbus/index.rst

```diff
@@ -1,58 +1,74 @@
 .. _zbus:
 
-Zephyr message bus (zbus)
-#########################
+Zephyr Bus (ZBus)
```

Comment:
If you want to spell it ZBus from now on please also consider updating other places where it might be referred to as "zbus", for example in code samples. FWIW I liked the lower-case naming :)